packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -101,6 +109,16 @@ function getMultiChainFetchList(): ChainPair[] {
     const childChainIds = chain.partnerChainIDs ?? []
     const isParentChain = childChainIds.length > 0
 
+    // for considering teleport (L1-L3 transfers) we will get the L3 children of the L1 chain
+    if (isL1Chain(chain)) {
+      const teleportAllowList = TELEPORT_ALLOWLIST[chain.chainID] || []
+      teleportAllowList.forEach(chainId => {
+        if (chainId && !childChainIds.includes(chainId)) {
```

Comment:
`chainId` should always be defined here I think

Reply:
Updated